src/Resolution.ts

```diff
@@ -486,13 +489,30 @@ export default class Resolution {
     return await this.getNamingMethodOrThrow(domain).allRecords(domain);
   }
 
+  async dns(domain: string, types: DnsRecordType[]): Promise<DnsRecord[]> {
+    const dnsUtils = new DnsUtils();
+    domain = this.prepareDomain(domain);
+    const method = this.getNamingMethodOrThrow(domain);
+    const dnsRecordKeys = this.getDnsRecordKeys(types);
+    const blockchainData = await method.records(domain, dnsRecordKeys);
+    return dnsUtils.toList(blockchainData);
```

Comment:
@bogdan @mvlabat @JohnnyJumper Should we return a list format here? Maybe let's return the raw results and make our DnsUtils class public to end-users?